### PR TITLE
Update package.json for explicit dependencies

### PR DESCRIPTION
**Pull request by terakilobyte**
Explicitly includes npm dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,6 +122,7 @@
     "yargs": "^3.30.0"
   },
   "devDependencies": {
+    "babel-core": "^5.8.34",
     "blessed": "~0.1.81",
```
Comment (BerkeleyTrue) [inline package.json]: babel-core is already included above in deps

**Comment by BerkeleyTrue** [comment]
What issue is this PR trying to solve?

**Comment by terakilobyte** [comment]
Taken from https://docs.npmjs.com/files/package.json

> NOTE: npm versions 1 and 2 will automatically install peerDependencies if they are not explicitly depended upon higher in the dependency tree. In the next major version of npm (npm@3), this will no longer be the case. You will receive a warning that the peerDependency is not installed instead.

I included the dependencies that were still required after running npm install with no node_modules directory. My npm version is 3.3.12

**Comment by BerkeleyTrue** [comment]
Those are already explicitly required. You've just update them.

**Comment by BerkeleyTrue** [comment]
@ltegman Are you having the same difficulties? You're last PR only required react-dom.

By the way: We are only officially supporting Node 4/npm 2 until Node 6 is released.

**Comment by ltegman** [comment]
@BerkeleyTrue react-dom was the only dependency I had to manually install when I used npm 3. It also warned me about kerberos for mongodb, but we don't use kerberos.

**Comment by BerkeleyTrue** [comment]
@ltegman thanks for the confirmation.

Closing this. @terakilobyte npm may have had a hiccup during install.

**Pull request closed without merging**